code/pom.xml

```diff
@@ -8,6 +8,8 @@
   <version>1</version>
```

Comment:
High Level comment. 

1) This PR is humongous, many of these files could be their own PR. Don't bother breaking it up now, I think, but keep this in mind for future PR. 
2) Please test.